src/ucs/sys/string.c

```diff
@@ -345,14 +345,15 @@ ssize_t ucs_path_calc_distance(const char *path1, const char *path2)
     for (i = 0; i < comp_len; i++) {
         if (resolved_path1[i] != resolved_path2[i]) {
             same = 0;
+	    break;
```

Comment:
1. fix indent
2. this should be the only change in the PR. maybe except documentation

Reply:
@yosefe will fix indent but we also need the following changes: https://github.com/openucx/ucx/pull/6174/files#diff-0b6515c4e7e67fcf8cef62fd18e3e0c4a15ad1f59b0bdb85513542e8dbccef9dL140-L149
These changes will make sure that we can prioritize between two devices which are both at a path_distance > 4.